plugins/modules/jenkins_plugin.py

```diff
@@ -112,7 +124,8 @@
   with_dependencies:
     description:
       - Defines whether to install plugin dependencies.
-      - This option takes effect only if the O(version) is not defined.
+      - In earlier versions, this option had no effect when a specific C(version) was set.
+      - Since community.general 11.1.0, dependencies are also installed for versioned plugins.
```

Comment:
This should be part of the same paragraph as the previous line.
```suggestion
        Since community.general 11.1.0, dependencies are also installed for versioned plugins.
```